pycontrails/models/accf.py

```diff
@@ -132,47 +154,34 @@ class ACCF(Model):
     sur_variables = (ecmwf.SurfaceSolarDownwardRadiation, ecmwf.TopNetThermalRadiation)
     default_params = ACCFParams
 
-    short_vars = [v.short_name for v in met_variables + sur_variables]
+    short_vars = {v.short_name for v in (*met_variables, *sur_variables)}
 
-    ds_met: xr.Dataset | None
-    ds_sur: xr.Dataset | None
+    # This variable won't get used since we are not writing the output
+    # anywhere, but the library will complain if it's not defined
+    path_lib = "./"
 
     def __init__(
         self,
         met: MetDataset,
         surface: MetDataset | None = None,
-        params: dict[str, Any] = {},
+        params: dict[str, Any] | None = None,
         **params_kwargs: Any,
     ) -> None:
         # Normalize ECMWF variables
         met = standardize_variables(met, self.met_variables)
 
-        if surface:
-            surface = standardize_variables(surface, self.sur_variables)
-
-        # Surpress warning about humdity scaling because that should be eet
-        # using ACCF config variables for this model
-        try:
-            del met.attrs["history"]
-        except KeyError:
-            pass
-
-        source = met.attrs["met_source"]
-        met.attrs["met_source"] = "not_ecmwf"
-        super().__init__(met, params=params, **params_kwargs)
-        if self.met:
-            self.met.attrs["met_source"] = source
+        # Ignore humidity scaling warning
+        with warnings.catch_warnings():
+            warnings.filterwarnings("ignore", module="pycontrails.core.models")
+            super().__init__(met, params=params, **params_kwargs)
 
-        self._update_accf_config()
         if surface:
-            self.surface = surface.copy()
-
-        # This variable won't get used since we are not writing the output
-        # anywhere, but the library will complain if it's not defined
-        self.path_lib = "./"
+            surface = surface.copy()
+            surface = standardize_variables(surface, self.sur_variables)
+            surface.data = _rad_instantaneous_to_accumulated(surface.data)
```

Comment:
I think we can restrict ACCF usage to ECMWF ERA5 or HRES and then we can skip the `_rad_instantaneous..` step.

Or are you tryting to handle a different circumstnace here? The bug in #64 is due to us converting J/m2 to W/m2, not the other way around.

Reply:
I think we probably should leave this `_rad_instantaneous_to_accumulated` function. If anyone has previously cached ERA5 or HRES radiation data with pycontrails (for example, us), then it will have radiation expressed in W/m2. And after we merge this branch, any newly cached data will have radiation expressed in J/m2. So this handles both cases by inspecting the unit.

Reply:
Ah right, good point.